disassemblers/ofrak_pyghidra/ofrak_pyghidra/components/pyghidra_components.py

```diff
@@ -108,8 +109,11 @@ async def analyze(self, resource: Resource, config: PyGhidraAutoAnalyzerConfig =
                     return PyGhidraProject()
 
             program_attrs = resource.get_attributes(ProgramAttributes)
-            code_regions = await resource.get_children_as_view(CodeRegion)
-            base_address = min(code_region.virtual_address for code_region in code_regions)
+            regions = await resource.get_children_as_view(
+                MemoryRegion, r_filter=ResourceFilter.with_tags(MemoryRegion)
+            )
+            base_address = min(code_region.virtual_address for code_region in regions)
```

Comment:
```suggestion
            # Guess that the base address is the min start address of any memory region
            regions = await resource.get_children_as_view(
                MemoryRegion, r_filter=ResourceFilter.with_tags(MemoryRegion)
            )
            base_address = min(code_region.virtual_address for code_region in regions)
```

I think a clarifying comment here is important